ibm/service/power/resource_ibm_pi_network_security_group_member.go

```diff
@@ -0,0 +1,375 @@
+							Type:        schema.TypeString,
+						},
+						Attr_Protocol: {
+							Computed:    true,
+							Description: "The list of protocol.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									Attr_ICMPTypes: {
+										Computed:    true,
+										Description: "If icmp type, the list of ICMP packet types (by numbers) affected by ICMP rules and if not present then all types are matched.",
+										Elem: &schema.Schema{
+											Type: schema.TypeInt,
+										},
+										Type: schema.TypeList,
+									},
+									Attr_TCPFlags: {
+										Computed:    true,
+										Description: "If tcp type, the list of TCP flags and if not present then all flags are matched.",
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												Attr_Flag: {
+													Computed:    true,
+													Description: "TCP flag.",
+													Type:        schema.TypeString,
+												},
+											},
+										},
+										Type: schema.TypeList,
+									},
+									Attr_Type: {
+										Computed:    true,
+										Description: "The protocol of the network traffic.",
+										Type:        schema.TypeString,
+									},
+								},
+							},
+							Type: schema.TypeList,
+						},
+						Attr_Remote: {
+							Computed:    true,
+							Description: "List of remote.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									Attr_ID: {
+										Computed:    true,
+										Description: "The ID of the remote Network Address Group or network security group the rules apply to. Not required for default-network-address-group.",
+										Type:        schema.TypeString,
+									},
+									Attr_Type: {
+										Computed:    true,
+										Description: "The type of remote group the rules apply to.",
+										Type:        schema.TypeString,
+									},
+								},
+							},
+							Type: schema.TypeList,
+						},
+						Attr_SourcePort: {
+							Computed:    true,
+							Description: "List of source port",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									Attr_Maximum: {
+										Computed:    true,
+										Description: "The end of the port range, if applicable. If the value is not present then the default value of 65535 will be the maximum port number.",
+										Type:        schema.TypeInt,
+									},
+									Attr_Minimum: {
+										Computed:    true,
+										Description: "The start of the port range, if applicable. If the value is not present then the default value of 1 will be the minimum port number.",
+										Type:        schema.TypeInt,
+									},
+								},
+							},
+							Type: schema.TypeList,
+						},
+					},
+				},
+				Optional: true,
+				Type:     schema.TypeList,
+			},
+			Attr_UserTags: {
+				Computed:    true,
+				Description: "The user tags associated with this resource.",
```

Comment:
```suggestion
				Description: "The user tags attached to this resource.",
```

This change is to match the wording in IBM Cloud. May need to be updated throughout.

Reply:
We don't need it for right now. Removed.